src/App.js

```diff
@@ -47,6 +48,9 @@ function App() {
           <PrivateRoute path="/lists/:id">
             <List />
           </PrivateRoute>
+          <PrivateRoute path="/aboutUs">
```

Comment:
hey, let's take it out of the private route as people will want to know more about the app before they sign up; also, can we change the path to just "/about" ?